modules/lang-painless/src/main/java/org/elasticsearch/painless/PainlessScriptEngine.java

```diff
@@ -340,7 +341,7 @@ public Loader run() {
         GeneratorAdapter deterAdapter = new GeneratorAdapter(Opcodes.ASM5, isResultDeterministic,
             writer.visitMethod(Opcodes.ACC_PUBLIC, methodName, isResultDeterministic.getDescriptor(), null, null));
         deterAdapter.visitCode();
-        deterAdapter.push(scriptRoot.deterministic);
+        deterAdapter.push(scriptRoot.isDeterministic());
```

Comment:
What's happening here?

Reply:
I moved ScriptRoot to a different package. This required a getter for a package-private member resulting in `isDeterministic`.